modules/claims_api/spec/factories/auto_established_claims.rb

```diff
@@ -68,4 +68,42 @@
       evss_response { 'something' }
     end
   end
+
+  factory :bsg_response, class: OpenStruct do
+    bnft_claim_dto { create(:bnft_claim_dto).to_h }
```

Comment:
You can use an association here. https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#associations

Reply:
also, `bsg`? assuming that's a typo.